### PR TITLE
feat(inventory): memory model import

### PR DESCRIPTION
**Pull request by po1vo**
The import had been ignoring memory models.

| Q             | A
| ------------- | ---
| Bug fix?      | probably
| New feature?  | yes/no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A

### CODE REVIEW

**Review by stonebuzz — APPROVED** [review]
Seems good for me (on ```add``` and on ```update```)

File: src/CommonDevice.php [hunk 1]
```diff
@@ -495,6 +495,9 @@ public function import(array $input)
                         ];
                         break;
                 }
+            } else if (preg_match('/^.+models_id/', $field)) {
```
Comment (cedric-anne) [inline src/CommonDevice.php:498]: Maybe this `null` condition should be added on all fields.

Comment (po1vo) [reply src/CommonDevice.php:498]: No, sometimes an item just misses some data, so we should update it, not create a new one.
I tried adding `null` on all fields and it broke a lot of tests.

**Review by cedric-anne — APPROVED** [review]

**Comment by cedric-anne** [comment]
> While we're at it i suggest changing memory "designation" from `$type - $description` to `$type-$frequence`, so `DDR4 - SODIMM` becomes `DDR4-2133` which is IMO more descriptive.

Please open another PR for it.

**Review by trasher — APPROVED** [review]

**Pull request merged**